04-ruby-mvc-sinatra/sinatra-rest/readme.md

```diff
@@ -136,25 +170,60 @@ get '/:num_bottles' do
 end
 ```
 
+> Variables that we want to use in the view `.erb` files need to be instantiated with `@`.
+
 ```html
 <!-- /views/index.erb -->
 <%= @num_bottles %> of beer on the wall.
 <a href='/<%= @next %>'>Take one down.</a>
 ```
 
+> We can write erb tags in two ways. `<% %>` or `<%= %>`. Without the equal sign, the view (`.erb` file) will execute the code only. With an equal sign, the view will execute the code and also place a string that is the return value of the code that was executed.
+
+To see this in action let's update our `myapp.rb`:
+
+```ruby
+get '/' do
+  @bob = "bobert"
+  erb :index
+end
+```
```

Comment:
i know students will ask about scope and how `@bob` is not accessible to other routes. maybe mention why that is? a la event driven programming?